cfpb/consumerfinance.gov
# Refactor analytics to use canonical @cfpb/cfpb-analytics module

Fixes https://github.com/cfpb/consumerfinance.gov/issues/2361

## Changes

- ~~Change the analytics to lazy initialize when the `sendEvent` method is called.~~
- Updates codebase to use @cfpb/cfpb-analytics module.


## How to test this PR

1. ~~Put a `console.log(…)` statement in the init method of the Analytics.js module and `xdescribe` the Analytics-spec tests to skip them and run `yarn jest test/unit_tests/js/modules/Analytics-spec.js` and the console.log should not run. It will run on `main`, showing that the module calls `init()` when it it imported.~~
1. Add `debug-gtm=true` to a URL that fires analytics events and check the console.log. For example, http://localhost:8000/about-us/blog/?debug-gtm=true
